src/pages/game/guess-bar/guessSlice.ts

```diff
@@ -1,23 +1,51 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 
 import { RootState } from '../../../app/store';
+import { guessStateType } from '../../../types/guessStateType';
+import { POLISH_COMMON_WORDS } from '../../../utils/constants';
 
-const initialState = {
-    text: '',
+const initialState: guessStateType = {
+    guessedWords: [],
```

Comment:
its not needed - use guessHistory which is empty now so you can mock it if you wish

Reply:
![image](https://user-images.githubusercontent.com/74867348/196907665-89593807-a800-47cc-8f83-552b7b46bb3f.png)

I mock it like that its working as intended and I think its ok for now.